source/CardGames.py

```diff
@@ -153,28 +152,28 @@ def resetDeck(self):
     self.deck.reset()
     self.deck.shuffle()
 
+#Split deck into 3 piles and escape_card/starting_cards
+def split_card():
+  game_deck=Deck()
```

Comment:
I think the function should take as a parameter a deck that has already been initialized in the main game loop and return the splits and starting hand